.eslintrc.json

```diff
@@ -0,0 +1,268 @@
+{
+    "extends": "eslint:recommended",
```

Comment:
@mgol eslint:recommended is very carefully managed preset, and any changes to it are exposed via semver.